downstream/titles/automation-dashboard/aap-comon

```diff
@@ -0,0 +1 @@
+../../aap-common/
```

Comment:
Delete the aap-comon symlink. You can use `rm aap-comon` because a symlink is just a file.